apps/auth/db/commands.py

```diff
@@ -27,13 +27,12 @@ class CreateUserCommand(superdesk.Command):
         superdesk.Option('--username', '-u', dest='username', required=True),
         superdesk.Option('--password', '-p', dest='password', required=True),
         superdesk.Option('--email', '-e', dest='email', required=True),
-        superdesk.Option('--admin', '-a', dest='admin', required=False),
+        superdesk.Option('--admin', '-a', dest='admin', required=False, default=False),
```

Comment:
ideally it should have also `action='store_true'` so that it can be used without any value, like `--admin`